schema_editor/app/scripts/resources/boundaries-service.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Boundaries($resource, Config) {
+        var urlString = Config.api.hostname + '/api/boundaries/:id';
+
+        var resource = {
+          'query': stdResource.query,
+          'get': stdResource.get,
+          'post': postWithFile,
+          'remove': stdResource.remove,
+          'delete': stdResource.delete
+        };
+
+        var stdResource = $resource(urlString, {id: '@uuid'}, {
+            query: {
+                method: 'GET',
+                transformResponse: function(data) { return angular.fromJson(data).results; },
+                isArray: true
+            }
+        });
+
+        return resource;
```

Comment:
Interesting. By abstracting $resource, this prevents you from doing something like:
```
function (Boundaries) {
    var newBoundary = new Boundaries({uuid: 'abcd', ... });
   newBoundary.save().then(function () { ... });
})
```

Which may or may not be a bit clearer, and would replace the `postWithFile` method (which I don't see but am assuming is just not implemented yet?).